spec/acceptance/user_manages_functions_spec.rb

```diff
@@ -34,4 +34,36 @@
     result = execute("SELECT * FROM test() AS result")
     expect(result).to eq("result" => "testest")
   end
+
+  it "handles functions with arguments" do
+    successfully "rails generate fx:function adder"
+    write_function_definition "adder_v01", <<-EOS
+      CREATE FUNCTION adder(x int, y int)
+      RETURNS int AS $$
+      BEGIN
+          RETURN $1 + $2;
+      END;
+      $$ LANGUAGE plpgsql;
+    EOS
+    successfully "rake db:migrate"
+
+    result = execute("SELECT * FROM adder(1, 2) AS result")
+    # require "byebug"
+    # byebug
+    puts "PG::VERSION: #{PG::VERSION.inspect}"
+    puts "ActiveRecord.gem_version: #{ActiveRecord.version.inspect}"
+    puts "result: #{result.inspect}"
+
+
+    # if PG::VERSION >= "0.18.0"
+    if ActiveRecord.version > Gem::Version.new("5.0.0")
+      expect(result).to eq("result" => 3)
```

Comment:
You can omit the conditional by enforcing the result to be either a string or an integer. I'd prefer that.